src/components/MainView.js

```diff
@@ -63,46 +63,44 @@ function MainView({ children }) {
             width: 100%;
           `}
         >
-          <Root.Provider>
+          <Root.Provider
+            css={`
```

Comment:
Had to include this change back because otherwise, the `ScrrollView` doesn't work. Maybe there is another way of doing it, so we can also prevent the console warning. What do you think @fabriziovigevani?

Reply:
Let´s keep it this way for now, we can address it later